department-of-veterans-affairs/caseflow
# Capitalize Veteran in add claimant modal

Resolves #14946

### Description
Capitalize Veteran in Add Claimant modal

### Acceptance Criteria

### Testing Plan
1. Switch to user with Mail Intake permission e.g. BVAISHAW
2. Enter date from past for 'Receipt Date'
3. Select any review option 
3. Answer 'Yes' to 'Is the claimant someone other than the Veteran'
4. Select 'Add Claimant'
5. Verify that Veteran is capitalized in the label for 'Claimant's relationship'

### User Facing Changes

 BEFORE|AFTER
 ---|---

<img width="880" alt="Screen Shot 2020-09-01 at 5 32 37 PM" src="https://user-images.githubusercontent.com/68879427/91919141-d8c1d100-ec79-11ea-9a76-6e7a03981187.png">
| 
<img width="1112" alt="Screen Shot 2020-09-01 at 5 29 48 PM" src="https://user-images.githubusercontent.com/68879427/91919159-e2e3cf80-ec79-11ea-9361-960fc5a55194.png">
